configure

```diff
@@ -138,6 +138,9 @@ while [ $# -ne 0 ]; do
     --disable-exceptions)
       append_cache_entry NO_EXCEPTIONS BOOL true
       ;;
+    --disable-arrow=*)
```

Comment:
```suggestion
    --disable-arrow)
```

Reply:
I had this changed locally already and forgot to add it to the index. It's fixed now. 🙄